appcelerator/titanium_mobile
# fix(android): Ti.UI.Picker "change" event not returning row titles from "selectedValue" as of 10.0.1

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-28460

**Summary:**
- Wrongly returning array of `[object Object]` strings instead of row titles.
- Regression as of 10.0.1. This issue was caught before release.

**Note:**
I caught this issue when testing [kitchensink-v2](https://github.com/appcelerator/kitchensink-v2). It has a "Picker" window where you can see this same issue.

**Test:**
1. Build and run `SpinnerSimpleTest.js` attached to [TIMOB-28460](https://jira.appcelerator.org/browse/TIMOB-28460) on Android.
2. Scroll through each column in the spinner.
3. Verify selected row titles are correctly shown in the log.
`@@@ Selected values: <Row1>,<Row2>,<Row3>`
